columnflow/tasks/external.py

```diff
@@ -269,6 +279,73 @@ def iter_nano_files(
 
             yield (lfn_index, input_file)
 
+    def _fetch_lfn_fallback(
+        self,
+        lfn: str,
+        selected_fs: str,
+        force: bool = False,
+    ) -> tuple[law.LocalFileTarget | None, os.stat_result | None, bool]:
+        """
+        Fetches an *lfn* via fallback mechanisms. Currently, only ``xrdcp`` for remote file systems
+        *selected_fs* with `root://` bases is supported. Unless *force* is *True*, no fallbacks are
+        performed in case they are not necessary in the first place (determined by the availability
+        of the ``gfal2`` package).
+
+        :param lfn: Logical file name to fetch.
+        :param selected_fs: Name of the file system to fetch the LFN from.
```

Comment:
It might make sense to explicitly state that the root protocol needs to be defined in the `selected_fs` in the law.cfg as either `base` or `base_filecopy`

Reply:
Added.